tests/unit_test/data_processor/action_serializer_test.py

```
@@ -0,0 +1,688 @@
+import os
+
+from mongoengine import connect
+
+from kairon import Utility
+from kairon.shared.actions.data_objects import Actions, HttpActionConfig, PyscriptActionConfig
+from kairon.shared.actions.models import ActionType, ActionParameterType, DbActionOperationType
+from kairon.shared.callback.data_objects import CallbackConfig
+from kairon.shared.data.data_validation import DataValidation
+
+os.environ["system_file"] = "./tests/testing_data/system.yaml"
```

Ruff

Use capitalized environment variable `SYSTEM_FILE` instead of `system_file`

Replace `system_file` with `SYSTEM_FILE`

(SIM112)